models/marts/rollups/agg_product_cat_monthly.yml

```diff
@@ -0,0 +1,19 @@
+version: 2
+
+models:
+  - name: agg_product_cat_monthly
+    description: 'One row per combination of product category and month,count the
+    number of order items, reviewed orders, count of 1 and 5 star reviews, 1 and 5
+    star review frequencies, median and average review scores. 
+    As well, total reviews, total 1 star reviews and total 5 star reviews are available
+    for the entire population for a given month in order to see how a given category compares'
+    tests:
+      - dbt_utils.expression_is_true:
+          expression:  (count_category_1_score + count_category_5_score) <= count_category_w_review
```

Comment:
this is a very interesting test. What's the rationale behind it? Do we assume 1s and 5s often have reviews?

Reply:
not too sure my thought process, I believe I thought all reviewed orders had scores. Will remove for now.